### DISCUSSION TITLE
Translation of Leah Rowe (libreboot) support letter

### DISCUSSION THREAD

**Discussion by amigdala999** [rms-support-letter/rms-support-letter.github.io#5563] [General]
Leah Rowe, libreboot founder, has [published](https://libreboot.org/news/rms.html) a support letter for RMS. She notes that

> **Translations wanted for this article. I want this news article to be available in as many languages as possible. Download the Markdown file here: [rms.markdown](https://notabug.org/libreboot/libreboot/src/master/www/news/rms.markdown) and then translate it into your language. Send the translated file to** leah@libreboot.org

If someone is working on such translation I suggest posting a note here to avoid doing the same work twice.

**Comment by 6r1d** [comment]
Thank you for pointing that out. I'm writing a russian translation, but before that, I'm finishing an article for a certain magazine. You seem to be from Russia, as well, contact me on letter's Discord server, so we can work together later, if you want to. :-)

**Please note**: there's a reference markdown document by Leah! It may be needed to check if your translation if very different from original content. Download it and use as a reference.

**Reply by Bluesboy** [reply]
One dude almost finished Russian translation

**Comment by nukeop** [comment]
It's fine to talk about it, but this isn't related to this repo, we are not affiliated.
